### PR TITLE
Added perpendicular_line and stop_on_black functions

### PR DESCRIPTION
**Pull request by Bir9**

### CODE REVIEW

File: blocks.py [hunk 1]
```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
         subtract = gyro.angle() - target  # subtracts color sensors
         multiply = subtract * SHARPNESS # multiplies the difference by 0.7
         tank.drive(SPEED, multiply) # follows the line
+        
+        
+def stop_on_black(left_cs, right_cs):
+        if left_cs.reflection() <= 14:
```
Comment (bashir2) [inline blocks.py:24]: We should check both sensors here:

```suggestion
        if left_cs.reflection() <= 14 and right_cs.reflection() <= 14:
```
we should also make 14 a constant.

File: blocks.py [hunk 2]
```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
...
+            return True
+        else:
+            return False
 
+        
+def perpendicular_line(tank, left_cs, right_cs, line_distance, left_wheel, right_wheel):
```
Comment (bashir2) [inline blocks.py:30]: Where do we use `left_wheel` and `right_wheel`?
```suggestion
def perpendicular_line(tank, left_cs, right_cs, line_distance):
```

File: blocks.py [hunk 3]
```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
...
+    if line_distance >= 0:
+        turn = 85
+    else:
+        turn = -85
+    tank.settings(200, 100, 100, 100)
```
Comment (bashir2) [inline blocks.py:35]: We should probably call `tank.stop()` or document that it is expected for the robot to be in stop position when this function is called.

Comment (Bir9) [reply blocks.py:35]: I think it better to document this as calling stop wouldn't be doing anything, will get to fixing some of these outliers tomorrow morning

**Pull request merged**